pkg/controller/install/install_controller.go

```diff
@@ -161,3 +162,36 @@ func autoInstall(c client.Client, ns string) (err error) {
 	}
 	return err
 }
+
+func addSCCforSpecialClusterRoles(u *unstructured.Unstructured) *unstructured.Unstructured {
+
+	// these do need some openshift specific SCC
+	clusterRoles := []string{
+		"eventing-broker-filter",
+		"knative-eventing-controller",
+		"in-memory-channel-controller",
+		"in-memory-channel-dispatcher",
+	}
+
+	matchesClusterRole := func(cr string) bool {
+		for _, i := range clusterRoles {
+			if cr == i {
+				return true
+			}
+		}
+		return false
+	}
+
+	// massage the roles that require SCC
+	if u.GetKind() == "ClusterRole" && matchesClusterRole(u.GetName()) {
```

Comment:
@jcrossley3 I added the closure, to have the function local to `addSCCforSpecialClusterRoles` - thx for suggestion!